alg/teca_component_area_filter.cxx

```diff
@@ -174,8 +179,33 @@ std::vector<teca_metadata> teca_component_area_filter::get_upstream_request(
         req.get("arrays", arrays);
     arrays.insert(labels_var);
 
-    // TODO -- remove the arrays we produce. this is only needed
+    // remove the arrays we produce. this is only needed
     // if the post-fix is set.
+    std::string var_post_fix = this->variable_post_fix;
```

Comment:
see comment in #184 , same code is used there, we can throw it in a utility library and reuse